### DISCUSSION TITLE
Cannot talk to Github from gopkg.in

### DISCUSSION THREAD

**Discussion by bhushan-mdn** [community/community#29332] [Other Feature Feedback, Questions, & Ideas] [answered]
Visiting [https://gopkg.in/yaml.v3](https://gopkg.in/yaml.v3),
```
Cannot obtain refs from GitHub: cannot talk to GitHub: Get https://github.com/go-yaml/yaml.git/info/refs?service=git-upload-pack: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
```
New here, don't know if this is the right place to ask for help. I just wanted to alert the Github Staff about this issue.

**Comment by matthewgoodman13** [comment, marked as answer]
Experiencing same issue and others are as well. Linking reddit post for visibility.
https://www.reddit.com/r/golang/comments/wmvt9v/is_gopkgin_down/

**Reply by bhushan-mdn** [reply]
Thanks a lot @matthewgoodman13! Mentioned this on [niemeyer/gopkg#63](https://github.com/niemeyer/gopkg/issues/63), thanks a lot for your help!
